README.md

```diff
@@ -19,7 +19,21 @@ This repo contains the sources of the app, netlify functions and configuration f
 * To test your code you can run:
     `npm run dev`
    
- By default, the front runs on `localhost:4200/`, and the back on `localhost:9000/`.
+ By default, the front runs on `localhost:4200/`, and the back on `localhost:4200/.netlify/functions/`.
+ 
+ ### 4 Unit Test :
+
+ To perform unit tests you have to run:
+    `npm run test`
+
+You can eventually want to change the configuration such as the [browser](https://karma-runner.github.io/3.0/config/browsers.html) used, this is done in [this file](https://github.com/Iteatime/Acrabadabra/blob/master/src/karma.conf.js).
```

Comment:
"Eventually" does not translate to "éventuellement" but to "finalement".
A good way to avoid mistakes is to write shorter and simpler text like : "Edit this file to change the configuration"

Reply:
About lambda testing : 

https://medium.com/vandium-software/unit-testing-aws-lambda-functions-in-node-js-7ad6c8f5000
https://serverless.com/framework/docs/providers/aws/guide/testing/